Flutter/pdf/loading-and-saving-document.md

```diff
@@ -50,3 +50,63 @@ List<int> bytes =await document.save();
 File('output.pdf').writeAsBytes(bytes);
 
 {% endhighlight %}
+
+## Save and open a PDF document in desktop
```

Comment:
Move this topic under the getting started section: 

![image](https://github.com/user-attachments/assets/b5c6bb2b-9920-4c5a-abf1-a1dec212dedd)

Reply:
Moved.